azurerm/resource_arm_network_interface.go

```diff
@@ -151,6 +151,13 @@ func resourceArmNetworkInterface() *schema.Resource {
 				Computed: true,
 			},
 
+			"private_ip_addresses": {
+				Type:     schema.TypeSet,
```

Comment:
I'm thinking this _may_ theoretically cause some troubles when referencing the field as set has computed indexes? 🤔

Reply:
Good point.. I'll look into switching this over to a list

Reply:
Updated